s3_project/tests/test_applicant_info_class.py

```diff
@@ -0,0 +1,37 @@
+from s3_project.classes.applicant_info_class import ApplicantInfoClean
+
+dict_1 = {'name': 'Shaun Blake', 'date': '01/02/2019', 'result': 'Pass', 'self_development': 'No', 'financial_support_self': 'Yes', 'geo_flex':'Yes'}
+dict_2 = {'name': 'Vincent van Goph', 'date':'23//01/2020', 'result': 'Fail', 'self_development': 'Yes', 'financial_support_self': 'No', 'geo_flex':'Yes'}
+dict_3 = {'name': 'Harry James Daniel Peter', 'date':'12//08//2012', 'result': 'Pass', 'self_development': 'Yes', 'financial_support_self': 'Yes', 'geo_flex':'Yes'}
+
+testing = ApplicantInfoClean()
+
+
+# Tests that the names are split up correctly
+def test_split_names():
+    assert testing.split_names(dict_1) == ['Shaun', 'Blake']
+    assert testing.split_names(dict_2) == ['Vincent van', 'Goph']
+    assert testing.split_names(dict_3) == ['Harry James Daniel', 'Peter']
+
+
+test_split_names()
```

Comment:
Are we not using pytest or unittest?  Why not?

Reply:
Pytest was not working for Alex so he used the function to test it.